src/AutoRest.CSharp/Common/Output/Models/Serialization/PropertySerialization.cs

```diff
@@ -33,23 +33,16 @@ internal abstract class PropertySerialization
         public CSharpType? SerializedType { get; }
 
         public bool IsRequired { get; }
-        public bool ShouldSkipSerialization { get; }
-        public bool ShouldSkipDeserialization { get; }
+        public bool ShouldExcludeInWireSerialization { get; }
```

Comment:
This name is confusing to me. "Should Exclude InWire serialization?" Could you elaborate it further?

Reply:
some background:
the serialization options has a `format` property, which accepts following format: wire (W), json (J) and xml (X)
This usually categorizes into wire and not wire.
Our previous system of serialization is always wire-serialization which serialize a thing and use the serialized result in the payload, that is why it is called "wire"
This property here simply mean "should exclude in wire serialization" where wire means the thing above.
When should a property be excluded in wire serialization? Usually when the property is readonly. We should not use the word `IsReadOnly` because only property could be readonly.